spec/models/event_spec.rb

```diff
@@ -23,62 +23,21 @@
 
 describe 'Event model', type: :model do
 
-  let(:event) { FactoryBot.build(:league) }
+  let(:event) { FactoryBot.build(:event) }
 
   it 'should not validate without name' do
-    league = FactoryBot.build(:league, name: nil)
-    expect(league.valid?).to eq(false)
+    event = FactoryBot.build(:event, name: nil)
+    expect(event.valid?).to eq(false)
   end
 
   it 'should not validate without discipline' do
-    league = FactoryBot.build(:league, discipline: nil)
-    expect(league.valid?).to eq(false)
+    event = FactoryBot.build(:event, discipline: nil)
+    expect(event.valid?).to eq(false)
   end
 
   it 'should not validate without game_mode' do
-    league = FactoryBot.build(:league, game_mode: nil)
```

Comment:
Why are you removing all these tests?

Reply:
event_spec shouldn't test things that are league and tournament specifics. So I extracted them into shared_examples see further down in the files list.

Reply:
Actually see further up in actual_events_example.rb ;)

Reply:
ok and where are the league specific tests that are not included in the shared_example?

Reply:
can you clarify? I'm pretty sure I got all of them in the shared example

Reply:
My question is if all the tests where general event tests or if there are any specific league or tournament tests left which you did not move into the shared example?

Reply:
Well there are specific league and tournament tests but they are in league_spec.rb and tournament_spec.rb like they are supposed to.